src/components/ebay-snackbar-dialog/examples/02-action/template.marko

```diff
@@ -0,0 +1,23 @@
+<ebay-snackbar-dialog open=state.open on-close('closeSnackbar')>
+    This is the action snackbar
+    <@actions>
+            <button on-click('closeSnackbar') accesskey="u" class="fake-link snackbar-dialog__cta">Undo<span class="clipped"> - Access Key: U</span></button>
```

Comment:
This should probably use ebay-button. Right @agliga?

We would also want to handle the addition of those classes ourselves.

I think we could also take care of appending the "Access Key: U" text ourselves. We would need a new `a11y-text-access-key` (default: "Access Key").

Reply:
hmm what would that look like exactly? Because if the user gives the button or whatever else they want there, how would we add the accesskey into the button the user is adding?

Reply:
@agliga If using ebay-button can we derive the accesskey char from the input? Or can we not do that for an attribute that is a straight pass through?

Reply:
Also, @austingardner look at toast-dialog for an example of what I mean for using ebay-button here.

Reply:
wait so we don't want the user to be able to pass whatever they want into the @action section? Like for example if they wanted an icon or something instead of the button

Reply:
It's always going to be an ebay-button

Reply:
okay so in this case we could just have the user pass in the button's text and then we could take care of everything else? we wouldn't need an @action in that case, just an @undo like @agliga said? and then that text between the @undo would be placed in the text the ebay-button

Reply:
Yes, we could do that. We could even do without the `@undo` and just use attributes of `action-text` and `action-access-key`. However I was hoping to keep the API somewhat consistent with toast-dialog. That would also allow more than one action button, if design team ever wanted.

Reply:
What do you guys think? Keep it similar to toast-dialog? Or go for something different and perhaps more streamlined?

Reply:
Yeah I like the `action-text` and `action-access-key` idea! I don't think we should let the user pass in something like `<button/>` if we're always going to use `<ebay-button/>`

Reply:
but at the same time yeah that's true that they might need more than one action button... hmmm...

Reply:
You can also have multiple attribute tags￼
they are passed in as an array. Look at my video changes with the source tag and you can see how it's done￼

Reply:
okay so if I have an action tag it could look something like this: 
```
<for|action| of=input.actions>
        <ebay-button class="fake-link snackbar-dialog__cta" on-click(action.on-click) >
            <${action.actionText}/>
            <span class="clipped"> - Access Key: ${action.action-access-key}</span>
        </ebay-button>
    </for>
```

But how would the on-click work? I don't know syntactically how to pass functions like that in Marko

Reply:
nvm about the function -> I just need to think Marko and not React